src/examples/zkapps/hello_world/run_live.ts

```diff
@@ -0,0 +1,93 @@
+// Live integration test against real Mina network.
+import { AccountUpdate, Field, Mina, PrivateKey, fetchAccount } from 'snarkyjs';
+import { HelloWorld, adminPrivateKey } from './hello_world.js';
+
+const useLocalNetwork = process.env.USE_LOCAL_NETWORK === 'true';
+const senderKey = useLocalNetwork
+  ? PrivateKey.fromBase58(
+      'EKEnVLUhYHDJvgmgQu5SzaV8MWKNfhAXYSkLBRk5KEfudWZRbs4P'
+    )
+  : PrivateKey.random();
+const sender = senderKey.toPublicKey();
+const zkAppKey = PrivateKey.random();
+const zkAppAddress = zkAppKey.toPublicKey();
+const transactionFee = 100_000_000;
+
+// Network configuration
+const network = Mina.Network(
+  useLocalNetwork
+    ? 'http://localhost:8080/graphql'
+    : 'https://proxy.berkeley.minaexplorer.com/graphql'
+);
+Mina.setActiveInstance(network);
+
+// Fee payer setup
+if (!useLocalNetwork) {
+  console.log(`Funding the Fee payer account.`);
+  await Mina.faucet(sender);
+}
```

Comment:
This is exactly what I'm planning of doing.
Currently there is a way to get accounts in [automated way](https://hub.docker.com/r/o1labs/mina-local-network) using Mina Accounts-Manager tool packaged with the Docker Image. 
Unfortunately in this PR I did not provide the client for it because it is second repo we need same helpers in.
zkapp-cli, for instance, has an ability to request an accounts dynamically. So should be possible for anyone else.
Hence, I was thinking to extract common helpers into separate library as a compliment to Dockerized local networks.
[Internal discussion](https://o1-labs.slack.com/archives/C028Q27R8UC/p1690873863306919).

Reply:
https://github.com/o1-labs/snarkyjs/issues/1074